public/js/home_page.js

```diff
@@ -0,0 +1,53 @@
+window.onscroll = function() {
+  navbarSticky()
+};
+
+// Get navbar
+var navbar = document.getElementById("navbar");
+
+// Get the offset position of the navbar
+var sticky = navbar.offsetTop;
+
+// Add the sticky class to the navbar when you reach its scroll position. Remove "sticky" when you leave the scroll position
+function navbarSticky() {
```

Comment:
Did you want the navigation to stay at the top of the page? I think you can use https://materializecss.com/navbar.html#navbar-fixed instead of this JS